docs/neon_os/On the Mark II

```diff
@@ -0,0 +1,27 @@
+# Neon OS on the Mark II
```

Comment:
Added file needs a `.md` extension. I'd also recommend replacing ` ` with `_`, so the path is URL-safe and we don't have to worry about URL-encoding things

Reply:
Done, also update the .yml file, and added all 3 things to a draft google doc how-to for documentation. :-)